### ISSUE TITLE
ILSpy doesn't work with voiceover screenreader on macOS

### ISSUE THREAD

**Issue by patricus3** [icsharpcode/ILSpy#3326]
hello.
I want to know, can you fix ILSpy so it works with all screenreader? it shouldn't be hard to do.
I want to use it but it's currently really annoying to use, since I have to use the VOCR app to read stuff and click on it

**Comment by christophwille** [comment]
macOS? Are you using https://github.com/icsharpcode/AvaloniaILSpy/?

**Comment by patricus3** [comment]
Oh, probably yes, because I have to use it cross platform, as far as I know avalonia sucks

**Comment by christophwille** [comment]
AvaloniaILSpy has been updated in ages (no maintainer), so it simply might be because the Avalonia build it uses is so old.

**Comment by patricus3** [comment]
Anything cross platform but good?
